method-proposals/PROPOSAL-did-cheqd.md

```diff
@@ -0,0 +1,142 @@
+| Established decentralized governance framework, with clear documentation, discussion forum and simple voting process. There have been 60 proposals on mainnet as of January 2025 |
+| **Usability: Simple implementation for developers** |
+| Yes, as evidenced by developers building across SDKs, APIs and Registrar implementations |
+| **Sustainability: Energy efficiency and eco-friendly infrastructure** |
+| While there is energy used in the running of a blockchain network, the maintainers have sought to offset carbon generated by validators running on network via [Regen Network](https://www.registry.regen.network/) |
+| **Economic Feasibility: DIDs costs of use must be reasonable** |
+| Yes, costs are set via the decentralized governance framework. Testnet is free to write to; and, mainnet is approximately $2 USD per DID create. |
+| **Legal Recognition: Cross-border frameworks for DID acceptance** |
+| Yes, supports key types approved by multiple national governments, as well as compliance in terms of the underlying asset to write to the network (e.g. MiCA compliant Electronic Money Tokens such as USDC). |
+| **Revocation and Recovery: Decentralized mechanisms for key rotation and DID recovery** |
+| Key rotation is inherent in the DID Method and is secure and verifiable, independent of DNS. Revocation is supported per the DID-Core specification. |
+| **Emerging Markets: Offline-friendly, low-bandwidth** |
+|   |
+| **Long-lived DIDs needed for long-lived VCs** |
+| Yes, DIDs and associated DID Documents, as well as past, present and future versions of these are immutably stored for use in long-lived VCs |
+| **Low and predictable marginal cost at scale (millions of accounts)** |
+| Yes, and price can be adjusted as scale increases accordingly. |
+| **Ability to create and update identifiers rapidly (within seconds)** |
+| Yes, support for 5000 to 10,000 transactions per second |
+| **Support for key rotation** |
+| Yes, with a verifiable history of the versions of the DID, past keys and rotations. |
+| **Reliable and predictable-latency operation, for updating and resolving** |
+| Yes. |
+| **Resolution should not require additional state or context** |
+| Yes. |
+| **DIDs are permanent and immutable account identifiers** |
+| Yes.   |
+| **Consider support for various DID Traits: https://identity.foundation/did-traits/** |
+| Yes, full suite of DID traits are identified and referenced. |
+| **Consider categories defined by DID Rubric: https://www.w3.org/TR/did-rubric/** |
+| Yes. |
+| **Who WANTS to standardize the DID method and commits to doing the work?** |
+| cheqd is able to fund the standardization of the method via its community pool, and the work will likely be carried out by the maintainers of the method |
+| **Are there AT LEAST two WG members who support standardization of a DID method?** |
+| Yes, cheqd and Danube Tech |
+| **Are there no trademark or IP issues?** |
```

Comment:
Would be good to be precise, something along the lines of: "To the best of our knowledge, `did:cheqd` does not infringe upon any existing intellectual property rights, patents, or trademarks.  If any unforeseen intellectual property or trademark concerns arise, Cheqd Foundation is committed to addressing them transparently and in good faith.  In the event that a legitimate claim is identified, Cheqd Foundation will assess the matter and, where appropriate, make a voluntary donation or take other reasonable actions to ensure continued open and equitable access."